SwiftGit2/Repository.swift

```diff
@@ -11,9 +11,46 @@ import Result
 
 public typealias CheckoutProgressBlock = SG2CheckoutProgressBlock
 
+private func SG2FetchOptions(credentials: Credential = .Default()) -> git_fetch_options {
```

Comment:
I'd remove the default here. It's always specified, and that's probably for the best.